lib/manager/composer/__snapshots__/artifacts.spec.ts.snap

```diff
@@ -73,7 +73,7 @@ Array [
 exports[`.updateArtifacts() returns null if unchanged 1`] = `
 Array [
   Object {
-    "cmd": "composer update --with-dependencies --ignore-platform-reqs --no-ansi --no-interaction --no-scripts --no-autoloader",
+    "cmd": "composer update --with-dependencies --ignore-platform-reqs --no-ansi --no-interaction",
```

Comment:
Is this related to the change?

Reply:
just to do some more code coverage. It's because i set the `trustLevel=high` on an existing test. 🙃